src/app.js

```diff
@@ -1 +1,31 @@
-// comment
+const express = require('express');
+
+require('dotenv').config();
+
+const connectToMongo = require('./db/connection');
+
+const authRoutes = require('./routers/auth');
+
+const itemRoutes = require('./routers/item');
+
+const userRoutes = require('./routers/user');
+
+const globalRoutes = require('./routers/global');
+
+const app = express();
+const port = process.env.NODE_LOCAL_PORT;
+
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+
+app.use('/api/auth', authRoutes);
+app.use('/api/items', itemRoutes);
+app.use('/api/users', userRoutes);
+app.use(globalRoutes);
```

Comment:
I think this should be `app.use('/api/global', globalRoutes);`

Reply:
Resolved.
P.S. In system architecture it is above api, so I thought I should not give it api endpoint.